apps/rewards/app/components/App/App.js

```diff
@@ -333,5 +333,18 @@ const EmptyContainer = styled.div`
 // eslint-disable-next-line react/display-name
 export default () => {
   const { api, appState, connectedAccount, displayMenuButton } = useAragonApi()
-  return <App api={api} {...appState} connectedAccount={connectedAccount} displayMenuButton={displayMenuButton} />
+  return (
+    <App
+      api={api}
+      rewards={appState.rewards}
+      myRewards={appState.myRewards}
+      metrics={appState.metrics}
+      myMetrics={appState.myMetrics}
+      amountTokens={appState.amountTokens}
+      claims={appState.claims}
+      {...appState}
+      connectedAccount={connectedAccount}
+      displayMenuButton={displayMenuButton}
```

Comment:
The newest version of the client does not need this, and the latest version of aragonAPI does not provide it. I think we can get rid of it.